foundry.toml

```diff
@@ -4,7 +4,7 @@ out = 'out'
 libs = ['lib']
 fs_permissions = [{ access = "read-write", path = "./release"}, { access = "read", path = "./test" }]
 gas_reports = ["*"]
-optimizer_runs = 2000
+optimizer_runs = 200
```

Comment:
speed up compile time and still avoids stack to deep errors

Reply:
2000 was already a concession. You should think of this  number as "how many times do I expect this contract to be called over its lifetime". If that number 10k+ then setting this to 200 will most likely substantially increase gas usage in the long run. We should just make sure we run gas comparisons and consider the tradeoffs before changing this number